watchtower-core/src/main/resources/templates/admin/encryption.html

```diff
@@ -134,9 +134,9 @@ <h6 class="card-title m-0">Current Data Encryption Keys</h6>
               <tr th:each="dek : ${deks}">
                 <td class="className" th:text="${dek.converterClassName}"></td>
                 <td th:text="${dek.rotationInProgress} ? 'Yes' : 'No'"></td>
-                <td th:text="${#temporals.format(dek.lastRotationDateTime, 'MMM d, YYYY h:mm a')} ?: 'Never'"></td>
+                <td class="localizetime" th:text="${dek.lastRotationDateTime != null ? dek.lastRotationDateTimeMillis: 'Never'}"></td>
                 <td th:if="${metadata.rotationScheduleEnabled}"
-                    th:text="${#temporals.format(dek.lastRotationDateTime.toLocalDate().atStartOfDay().plusDays(metadata.rotationPeriod), 'MMM d, YYYY h:mm a')}"></td>
+                    class="localizetime" th:text="${dek.lastRotationDateTime.toLocalDate().atStartOfDay().plusDays(metadata.rotationPeriod).toInstant(T(java.time.ZoneOffset).UTC).toEpochMilli()}"></td>
```

Comment:
Should this be a function in the metadata class? I know I wrote this code originally but it might be simpler than having all of this Java here